website_and_docs/content/blog/2024/selenium-vs-blog-posts.md

```diff
@@ -0,0 +1,36 @@
+
+However, some particular misconceptions regularly reappear across these blog posts.
+
+## It’s too hard to set up browsers and drivers compared to Playwright and Cypress
+
+This used to be true in the past as you had to download the drivers. This wasn’t too bad for GeckoDriver and SafariDriver as they could handle browser upgrades gracefully. Chromium based browsers on the other hand need you to update the drivers with every browser update.
+
+For over a year now, Selenium handles this automatically. If it can’t find a ChromeDriver or EdgeDriver, it will download it using [Selenium Manager](https://www.selenium.dev/blog/2022/introducing-selenium-manager/). Since its first release it has improved a lot and it is now probably the best in class since the latest versions of Selenium will even download browsers if it can and use that. Compared to Playwright and Cypress you don’t need to update your dependency on Selenium to update browsers and drivers, you still use the same browsers as your customers, and switching versions becomes a breeze: you don’t also have to change the test framework you’re using. And, let's not forget that it uses the browser that [Google recommends you use for testing](https://developer.chrome.com/blog/chrome-for-testing/).
+
+## Setting up a test runner is hard work where Playwright and Cypress have it built in…
+
+Well… maybe? Setting up E2E test frameworks with Selenium isn’t as difficult as some might suggest. The hard part really is making sure that the driver is in the right place and we’ve solved that as discussed above. Once that’s done, Selenium’s approach allows you to use whichever test runner you’re most comfortable with. If you’d like a “batteries included” approach, with Selenium tightly integrated with the test runner, then one of the many projects that use Selenium, such as [SeleniumBase, Nightwatch, Serenity, and so on](https://www.selenium.dev/ecosystem/#frameworks), might be the right tool for you.
+
+One thing to note is that Playwright is the only multi-language browser framework like Selenium. However, if you don’t use TypeScript or JavaScript you will still need to do the setup of the test framework yourself. Some testing frameworks have plugins that automatically set up the fixtures you might need. In the JavaScript/TypeScript space if you really need a built-in test runner there are downstream projects like NightwatchJS and tangential projects like WebdriverIO. Downstream projects use our libraries and tangential projects have their own libraries but still follow the WebDriver standard.
```

Comment:
This paragraph seems to muddle "playwright doesn't ship a test framework for all language bindings" with "some test frameworks have support for Selenium"